microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -69,25 +69,25 @@ Mobile devices have much less CPU power than desktops and laptops.  Whenever you
 
    ![CPU throttle.](../media/evaluate-performance-performance-capture-settings.msft.png)
 
-   When testing other pages; if you want to ensure that each page works well on low-end mobile devices, set CPU Throttling to **6x slowdown**.  The demo doesn't work well with 6x slowdown, so it just uses 4x slowdown for instructional purposes.
+   If you want to ensure that pages work well on low-end mobile devices, set CPU Throttling to **6x slowdown**.  The demo doesn't work well with 6x slowdown, so it just uses 4x slowdown for instructional purposes.
 
 
 ### Set up the demo
 
-It's hard to create a runtime performance demo that works consistently for all readers of the website.  The following section lets you customize the demo to make sure that your experience is relatively consistent with the screenshots and descriptions, regardless of your particular setup.
+The following section lets you customize the demo to make sure that your experience is relatively consistent with the screenshots and descriptions.
 
 1. Click the **Add 10** button until the blue icons move noticeably slower than before.  On a high-end machine, you can click it about 20 times.
 
 1. Click **Optimize**.  The blue icons should move faster and more smoothly.
 
 1. To better display a difference between the optimized and un-optimized versions, click the **Subtract 10** button a few times and try again.  If you add too many blue icons, you might max out the CPU, and then you might not observe a major difference in the results for the two versions.
 
-1. Click **Un-Optimize**.  The blue icons move slower and with more sluggishness again.
+1. Click **Un-Optimize**.  The blue icons should move slower and with more sluggishness again.
```

Comment:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/should-vs-must - "Don't use should to indicate probability."
Whether to say 'should' is somewhat a matter of technical judgment, of degree of likelihood.

Reply:
That's fine; we can stet this. Thanks.